modules/ztp-add-local-reg-for-sno-duprofile.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-advanced-policy-config.adoc
+
+:_module-type: CONCEPT
+[id="ztp-add-local-reg-for-sno-duprofile_{context}"]
+= Configuring the Image Registry Operator for local caching of images
+
+{product-title} manages image caching with a local registry. A remote application such as {sno} is subject to bandwidth restrictions when communicating with the centralized registry, which might result in long image download times.
```

Comment:
The term remote application is not quite correct here. We deploy applications in OpenShift. What about:

```suggestion
{product-title} manages image caching using a local registry. In edge computing use cases, clusters are often subject to bandwidth restrictions when communicating with centralized image registries, which might result in long image download times.
```

Reply:
Done.